src/NServiceBus.Core.Tests/Msmq/MsmqQueueCreatorTests.cs

```diff
@@ -9,16 +9,28 @@
     [TestFixture]
     public class MsmqQueueCreatorTests
     {
+
+        [TearDown]
+        public void TearDown()
+        {
+            DeleteQueueIfPresent("MsmqQueueCreatorTests.receiver");
+            DeleteQueueIfPresent("MsmqQueueCreatorTests.target1");
+            DeleteQueueIfPresent("MsmqQueueCreatorTests.target2");
+            DeleteQueueIfPresent("MsmqQueueCreatorTests.txreceiver");
+            DeleteQueueIfPresent("MsmqQueueCreatorTests.MsmqDefaultPermissions");
+            DeleteQueueIfPresent("MsmqQueueCreatorTests.NoChangesToExisting");
+            DeleteQueueIfPresent("MsmqQueueCreatorTests.permissions");
+            DeleteQueueIfPresent("MsmqQueueCreatorTests.too.long.B2568312E4BF424D8C875D0CD3F65A3E");
+            DeleteQueueIfPresent("MsmqQueueCreatorTests.badidentity");
+            DeleteQueueIfPresent(GetReallyLongQueueName());
```

Comment:
@timbussmann - if you look at my first commit, I started out with deleting the queues in each test. However, I also see @bording's point which is the test should be more about what its testing than the setup and teardown of the requirements of the test. A convention based approach is probably a little overkill. As it would mean querying all the list of private queues and then weeding out the ones not needed, etc and then deleting it. However, I do see your point. I will go over these tests and see if it really needs unique queue names for every single test.